utils/callData/registrationCalls.ts

```diff
@@ -119,7 +119,7 @@ function renew(
       durationInYears * 365,
       sponsor ?? 0,
       discountId ?? 0,
-      "0x" + metadataHash,
+      metadataHash,
```

Comment:
are you sure about this ? Why are you deleting the 0x now ?

Reply:
because in some function we do `"0x" + metadataHash` and then send the result to the calldata function, and sometime we send metadataHash and handle the addition of `0x` inside the function. I think we should do the same everywhere because it results in errors if you're not aware of this.

Reply:
To make sure of this I think we can type or verify that all metadataHash starts with `0x` by using this type

```ts
type HexString = `0x${string}`;
``` 

So if we forget, en error pops up :)

Reply:
great idea!